app/src/main/java/org/sagebionetworks/research/mpower/tracking/fragment/SelectionUIFormItemWidget.java

```diff
@@ -0,0 +1,54 @@
+package org.sagebionetworks.research.mpower.tracking.fragment;
+
+import android.content.Context;
+import android.support.annotation.ColorRes;
+import android.support.constraint.ConstraintLayout;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.TextView;
+
+import org.sagebionetworks.research.mpower.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class SelectionUIFormItemWidget extends ConstraintLayout {
```

Comment:
Check out Rian's current PR on how to do a custom view like this in Kotlin I think you will like it